app/jobs/cron/datagouv/base_job.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Cron::Datagouv::BaseJob < Cron::CronJob
+  include DatagouvCronSchedulableConcern
+
+  DATASET = '62d677bde7e4ca2c759142ce'
+  DATE_FORMAT = "%Y-%m"
+
+  def data_gouv_csv(resource, headers)
+    APIDatagouv::API.existing_csv(DATASET, resource) ||
+      CSV::Table.new([], headers:)
+  end
+
+  def missing_months(csv)
+    last_date = Date.strptime(csv[-1]['mois'], DATE_FORMAT) if csv.present?
```

Comment:
j'ai l'impression que le csv est toujours present car il est vien de la methode `data_gouv_csv`

```suggestion
    last_date = Date.strptime(csv[-1]['mois'], DATE_FORMAT)
```

Reply:
en effet!